src/core/library.c

```diff
@@ -99,13 +99,117 @@ MsQuicCalculatePartitionMask(
     MsQuicLib.PartitionMask = (HighBitSet << 1) - 1;
```

Comment:
Just happened to stumble upon this function and these 2 lines seem like unnecessary.

The result is exactly the same without these 2 lines except for zero, which is an undefined value in this context.